app/models/github_service.rb

```diff
@@ -9,4 +9,9 @@ def self.request_user(login)
     response = Faraday.get("https://api.github.com/users/#{login}?client_id=#{ENV['client_id']}&client_secret=#{ENV['client_secret']}")
     JSON.parse(response.body, symbolize_names: true)
   end
+
+  def self.get_repo(repo_name, login)
+    response = Faraday.get("https://api.github.com/repos/#{login}/#{repo_name}?client_id=#{ENV['client_id']}&client_secret=#{ENV['client_secret']}")
```

Comment:
we start each Faraday request with "https://api.github.com" => might be nice to make this a constant somewhere?

also - is there a reason we're sending the client_id and client_secret here rather than just the access_token? checkout the docs [here](https://developer.github.com/v3/#authentication) on what you should be passing to authenticate the request 👍